src/pages/RegisterPage/index.js

```diff
@@ -15,169 +15,158 @@ const RegisterPagePage = () => {
   const [username, setUsername] = useState("");
   const [country, setCountry] = useState("");
 
-  useEffect(() => {
-    if (supabaseClient.auth.user())
-      navigate("/listingspage");
-  });
-
```

Comment:
Should keep this code, or replace with code that redirects logged-in users out of the login/register pages.